hedera-node/hedera-app/src/main/java/com/hedera/node/app/records/impl/producers/formats/v6/BlockRecordWriterV6.java

```diff
@@ -390,6 +392,25 @@ private void closeSidecarFileWriter() {
         }
     }
 
+    /**
+     * Write a marker file for the sidecar file.
+     * This is used to indicate that the sidecar file has been closed and is ready to be uploaded.
+     * The marker file is named the same as the sidecar file, but with a .mf extension.
+     * The contents of the marker file are the hash of the sidecar file.
+     *
+     * @throws IOException if there was a problem writing the marker file
+     */
+    private void writeSidecarMarkerFile() throws IOException {
+        final Path sidecarPath = getSidecarFilePath(sidecarFileWriter.id());
+        final Path markerPath =
+                sidecarPath.resolveSibling(sidecarPath.getFileName().toString().replace(".rcd.gz", ".mf"));
+        if (Files.exists(markerPath)) {
+            logger.debug("Side‑car marker already exists: {}", markerPath);
```

Comment:
Definitely not a blocker, but maybe `.warn(...)` instead of `.debug(...)`

Reply:
Ah yes Matt, will change in next PR. Thanks